modules/service/src/main/scala/lucuma/odb/service/UserInvitationService.scala

```diff
@@ -53,8 +58,35 @@ object UserInvitationService:
     def apply(id: UserInvitation.Id, detail: Option[String]): OdbError.InvitationError =
       OdbError.InvitationError(UserInvitation.Id.fromString.reverseGet(id), detail)
 
-  def instantiate[F[_]: MonadCancelThrow](using Services[F]): UserInvitationService[F] =
+  def instantiate[F[_]: MonadCancelThrow](emailConfig: Config.Email, httpClient: Client[F])(using Services[F]): UserInvitationService[F] =
     new UserInvitationService[F]:
+      
+      // TODO: move to UserInvitation class?
+      extension(invitation: UserInvitation)
+        def token: String = s"${invitation.id}.${invitation.body}"
```

Comment:
Unless someone sees a reason not to, I'll move this to the UserInvitation in lucuma-core.